playgame.py

```diff
@@ -5,55 +5,111 @@
 from tools.hanabi_card import HanabiColor
 import sys
 import logging
+from logging.handlers import RotatingFileHandler
+import os
```

Comment:
What is `os` used for here? I don't see any instances of `os.`.

Reply:
Used to check if a file existed by hand to do a rotate. Need to remove